plugins/services/src/js/containers/service-connection/SDKServiceConnectionEndpointList.js

```diff
@@ -0,0 +1,286 @@
+import React from "react";
+
+import Loader from "#SRC/js/components/Loader";
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading
+  from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection
+  from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import Icon from "#SRC/js/components/Icon";
+import AlertPanel from "#SRC/js/components/AlertPanel";
+import AlertPanelHeader from "#SRC/js/components/AlertPanelHeader";
+import ClipboardTrigger from "#SRC/js/components/ClipboardTrigger";
+import RouterUtil from "#SRC/js/utils/RouterUtil";
+
+import Service from "../../structs/Service";
+import Pod from "../../structs/Pod";
+
+import SDKEndpointActions from "../../events/SDKEndpointActions";
+import SDKEndpointStore from "../../stores/SDKEndpointStore";
+import { EDIT } from "../../constants/ServiceActionItem";
+import ServiceActionDisabledModal
+  from "../../components/modals/ServiceActionDisabledModal";
+
+const METHODS_TO_BIND = [
+  "handleOpenEditConfigurationModal",
+  "handleActionDisabledModalClose",
+  "handleTextCopy"
+];
+
+class SDKServiceConnectionEndpointList extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      actionDisabledModalOpen: false,
+      copiedCommand: false
+    };
+
+    METHODS_TO_BIND.forEach(method => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  handleTextCopy(copiedCommand) {
+    this.setState({ copiedCommand });
+  }
+
+  handleOpenEditConfigurationModal() {
+    this.setState({
+      actionDisabledModalOpen: true
+    });
+  }
+
+  handleActionDisabledModalClose() {
```

Comment:
I would rather see we consolidate the setState for `actionDisabledModalOpen` in one method but receive the values as arguments. 
e.g `this.handleOpenEditConfigurationModal.bind(this, true)` and for `this.handleActionDisabledModalClose(this, false)`

What do you think?